docs/guides/http_clients.mdx

```diff
@@ -53,7 +53,7 @@ HttpClient --|> ImpitHttpClient
 
```

Comment:
Could we mention somewhere in this guide that our `HTTPX` client uses custom HTTP headers and fingerprints? That it is not just the bare `HTTPX`. We can also link the "Avoid getting blocked" guide.